lib/node_modules/@stdlib/ndarray/iter/to-array-each/README.md

```diff
@@ -158,6 +158,8 @@ while ( true ) {
 
 <!-- <related-links> -->
 
+
```

Comment:
@Planeshifter This seems extraneous. Is there a reason why two empty lines were added here?